### ISSUE TITLE
Questions about dataset preparing

### ISSUE THREAD

**Issue by ZihaoLinQZ** [zhang9302002/ThinkingWithVideos#4]
Hi,

Thanks for your excellent work!

I have several questions about the data preparation. When I tried to run `train_stage_1_sft.sh`, there are many datasets such as `data/charades/train_set_sft_new_10k6_src_diff.json`. However, I didn't find any file in charades whose name is `train_set_sft_new_10k6_src_diff.json` from their office link. And for other dataset such as `actnet`, `vidchapters`, etc, there are the same issues. 

Could you please provide a detailed instruction how to construct the dataset, including how to rename/move the files downloaded from other datasets to correct path in your setting?

Thanks!

**Comment by CR400AF-A** [comment]
same question

**Comment by zhang9302002** [comment]
Thanks for your pointing out.

We are sorry for the misleading of training script. We organized our training dataset here: https://huggingface.co/datasets/zhang9302002/MultiTaskVideoReasoning

You can revise the training script to adapt to the hf dataset.

We will correct the training script soon.

**Comment by CR400AF-A** [comment]
Thank you for your work. 

I have downloaded train dataset, but the sft scripts still need `dataset_json_path_val` and `test_set` for evaluation (required by `eval_temporal_grounding.py`). Could you please upload them?

Besides, what does `video_14400frames_fps2` mean?

**Comment by zhang9302002** [comment]
Good suggestion. We will upload these test files in the next update. You are also welcome to format the original annotation from these benchmarks.

That means all frames are extracted with FPS=2, and max_frames_per_video=14400. If a video is longer than 2 hours, we will uniformly sample 14400 frames with FPS < 2. (in very rare cases)

**Comment by Lwt235** [comment]
> Good suggestion. We will upload these test files in the next update. You are also welcome to format the original annotation from these benchmarks.
> 
> That means all frames are extracted with FPS=2, and max_frames_per_video=14400. If a video is longer than 2 hours, we will uniformly sample 14400 frames with FPS < 2. (in very rare cases)

Hi, thanks a lot for releasing this code.

I noticed that some video files in vidchapters.json have a suffix like `.f243.mp4` (e.g., `RA6g_KwF5QY.f243.mp4`). 
May I ask whether `f243` indicates a preprocessed version of the video with a fixed number of sampled frames (e.g., 243 uniformly sampled frames for model input)?

I just want to make sure I correctly understand the video preprocessing protocol and stay consistent with the provided setup.

Thanks in advance for your clarification!

**Comment by Lwt235** [comment]
I also noticed that the annotations in `vidchapter.json` (e.g., fields like id, tool_params, duration, text, etc.) do not include any audio transcript—how is the audio modality handled in your setup?

**Comment by re-burn** [comment]
> Good suggestion. We will upload these test files in the next update. You are also welcome to format the original annotation from these benchmarks.
> 
> That means all frames are extracted with FPS=2, and max_frames_per_video=14400. If a video is longer than 2 hours, we will uniformly sample 14400 frames with FPS < 2. (in very rare cases)

Any updates about the valid set and test set?
